numba/testing/main.py

```diff
@@ -639,13 +639,21 @@ def _run_inner(self, result):
         # method as if it were a test case.
         child_runner = _MinimalRunner(self.runner_cls, self.runner_args)
 
-        # Split the tests and recycle the worker process to tame memory usage.
-        chunk_size = 500
-        splitted_tests = [self._ptests[i:i + chunk_size]
-                          for i in range(0, len(self._ptests), chunk_size)]
-
-        for tests in splitted_tests:
-            pool = multiprocessing.Pool(self.nprocs)
+        if six.PY2:
+            # Split the tests and recycle the worker process to tame memory usage.
+            chunk_size = 500
+            splitted_tests = [self._ptests[i:i + chunk_size]
+                              for i in range(0, len(self._ptests), chunk_size)]
+            target = splitted_tests
+        elif six.PY3:
+            target = [self._ptests]
+        t = time.time()
+        for tests in target:
+            if six.PY2:
+                pool = multiprocessing.Pool(self.nprocs)
+            elif six.PY3:
```

Comment:
Perhaps this branching needs to be based on  `< 3.4` as that was when `.get_context` was added?

Reply:
yes, changed and pushed.